code/go/0chain.net/sharder/m_handler_test.go

```diff
@@ -208,9 +211,7 @@ func TestNotarizedBlockKickHandler(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
-		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
These tests should be done in parallel as other errors might be missed. In practice we can expect 0chain to handle multiple HTTP test at once, using t.Parallel is an easy way to simulate that.